src/scripts/site.js

```diff
@@ -404,10 +402,6 @@ var AppInfo = {};
         define('localassetmanager', [bowerPath + '/apiclient/localassetmanager'], returnFirstDependency);
```

Comment:
why this was not changed?

Reply:
Oversight when merging master, fixed in latest commit